src/translations/translations.fr.js

```diff
@@ -255,9 +255,8 @@ export const translations = {
   'cc-domain-management.certif.custom': () =>
     sanitize`Vous pouvez fournir votre propre certificat grâce au <a href="https://api.clever-cloud.com/v2/certificates/new">gestionnaire de certificats Clever Cloud</a>.`,
   'cc-domain-management.certif.heading': `Sécurisez votre application`,
-  'cc-domain-management.dns.a.desc':
-    () => sanitize`<p>Si vous choisissez d'utiliser des enregistrements de type <code>A</code>, vous devrez vous-même assurer leur mise à jour.</p>
-  <p>Pensez à suivre notre <a href="https://developers.clever-cloud.com/changelog/" lang="en">changelog</a> ou à utiliser notre <a href="https://developers.clever-cloud.com/api/v4/#load-balancers" lang="en">API v4</a> pour cela.</p>`,
+  'cc-domain-management.dns.a.desc': () =>
+    sanitize`<p>Si vous choisissez d'utiliser des enregistrements de type <code>A</code>, vous devrez vous-même assurer leur mise à jour. Pensez à suivre notre <a href="https://developers.clever-cloud.com/changelog/" lang="en">changelog</a> ou à utiliser notre <a href="https://developers.clever-cloud.com/api/v4/#load-balancers" lang="en">API v4</a> pour cela.</p>`,
```

Comment:
nitpick: In english we used `v4 API documentation`, but there's no mention of the word `documentation` word here. Maybe we could add it to get some clarity?

Reply:
This would make the line break to a new line with only one or two words which is why David removed it. I think it's fine, it's a little bit clearer with "documentation" but it's not bad like this so let's keep it.